apps/src/templates/rubrics/rubricShapes.jsx

```diff
@@ -36,7 +36,7 @@ export const reportingDataShape = PropTypes.shape({
 
 export const studentLevelInfoShape = PropTypes.shape({
   name: PropTypes.string,
-  timeSpentMinutes: PropTypes.number,
+  timeSpent: PropTypes.number,
```

Comment:
Fixing for type inference purposes - I think this is actually `timeSpent` within rubrics code, for example:
https://github.com/code-dot-org/code-dot-org/blob/6462579421261a11e8236f718763462502b8232a/dashboard/app/controllers/script_levels_controller.rb#L226